src/main/User/UserController.java

```diff
@@ -217,7 +217,24 @@ public Handler loginUser(SecurityUtils securityUtils, EmailUtil emailUtil) {
         ctx.json(res.put("username", candidateUsername).toString());
         logger.info("Username successfully generated");
       };
-
+  public Handler usernameExists =
```

Comment:
I think you left this in :) 
If you want though, if you write a service class for this, that would be great.